### ISSUE TITLE
FNX-4998 ⁃ Site info/permissions button has no label

### ISSUE THREAD

**Issue by eeejay** [mozilla-mobile/fenix#1392]
![image](https://user-images.githubusercontent.com/5764/55588641-216f0680-56e3-11e9-8499-f03d4014f4aa.png)

**Comment by vesta0** [comment]
@mheubusch will provide strings for the label.

**Comment by mheubusch** [comment]
String for alt text: Show site information

**Comment by eeejay** [comment]
@mheubusch how do you feel about "Site information"? Brevity is important for screen readers. The user will hear "Site information, button", which indicates it is interactive.

**Comment by ekager** [comment]
The work for this will be done in AC.

**Comment by AndiAJ** [comment]
Hi, **verified as fixed** on the latest **Nightly Build #13050627 from 1/11** using the following devices:
• Google Pixel 3a XL (Android 9)
• Huawei Mate 20 Lite (Android 8.1.0)
• Samsung Galaxy S7 (Android 7)
